### PR TITLE
Booking controller setup

### PR DESCRIPTION
**Pull request by AnthonyHad**
Bookings Controller 
Index + View
Show + View
New / Create + View

### CODE REVIEW

File: app/views/bookings/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<h1>Your Pending Bookings</h1>
+
+<ol>
+  <% @bookings.each do |booking|  %>
+  <li><%= booking.start_date %> || <%= booking.end_date %> || <%= booking.status%></li>
+  <h3><%=link_to booking.equipment.name, booking_path(booking)%></h3>
```
Comment (Nobaltha) [inline app/views/bookings/index.html.erb:6]: Why is it a h3?

File: app/views/bookings/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h1>Booking by <%=@booking.user.first_name%></h1>
+
+<ul>
+  <li><%= @booking.start_date %> || <%= @booking.end_date %> || <%= @booking.status%></li>
+  <h3><%= @booking.equipment.name%></h3>
```
Comment (Nobaltha) [inline app/views/bookings/show.html.erb:5]: Why is it a h3 as well?

File: app/views/bookings/index.html.erb (hunk 1 above)
Comment (AnthonyHad) [reply app/views/bookings/index.html.erb:6]: Just to keep it not so big and not so small!

File: app/views/bookings/show.html.erb (hunk 2 above)
Comment (AnthonyHad) [reply app/views/bookings/show.html.erb:5]: Just to keep it not so big and not so small!

**Pull request merged**